content/docs/concepts/traffic-management/virtual-services/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Virtual Services
+description: Describes the architecture and behavior of virtual services and provides configuration examples.
+weight: 3
+keywords: [routing, request, traffic routing, requests routing, virtual service, subset, routing rules]
+aliases:
+---
+
+A [virtual service](/docs/reference/config/istio.networking.v1alpha3/#VirtualService)
+configures the rules controlling how **Pilot** routes requests for a service
+within an Istio service mesh. A virtual service contains an ordered list of
+[routing rules](../routing-rules/).
+
+You can use virtual services for fine-grained traffic control. Istio can use
+virtual services to configure ingress, canary, traffic rewrite, or traffic
+policing. A virtual service is not required for simple connectivity. A service
```

Comment:
You can use virtual services for fine-grained traffic control, including configuring ingress traffic and routing to specific service versions. However, you don't require a virtual service for simple connectivity.